documentation/modules/cruise-control/ref-cruise-control-configuration.adoc

```diff
@@ -58,14 +58,15 @@ spec:
     # ...
 ----
 
-[#capacity-configuration]
+[[capacity-configuration]]
 [discrete]
 == Capacity configuration
 
-Cruise Control uses _capacity limits_ to determine if optimization goals for broker resources are being broken.
-An optimization will fail if a hard goal is broken, preventing the optimization from being used to generate an optimization proposal.
+Cruise Control uses _capacity limits_ to determine if certain resource-based optimization goals are being broken. 
+An attempted optimization fails if one or more of these resource-based goals is set as a hard goal and then broken. 
```

Comment:
So this is actually a bit redundant as any goal, resource based or not, will fail the optimization if it set as hard and then not met. You could just remove this sentence and the one after and it would be just as accurate and much less confusing.